<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1507</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(15,15,15);
color:rgb(202,202,202);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="main_tab_widget">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane{
        border: 2px solid rgb(55,55,55);
}
 
QTabBar::tab{
        border: 2px solid rgb(55,55,55);
        border-top-left-radius: 4px;
        border-top-right-radius: 4px;
        min-width: 8ex;
        padding: 2px;
}
 
QTabBar::tab:selected{
        border-bottom-color:rgb(55,55,55);
}
 
QTabBar::tab:!selected{
        margin-top: 4px;
}
 
QSpinBox{
        background-color:rgb(55,55,55);
}
 
QPushButton{
        background-color:rgb(55,55,55);
}

QLineEdit{
	border: 1px solid rgb(55,55,55);
}

QTimeEdit{
	border: 1px solid rgb(55,55,55);
}

QCheckBox:indicator{
	background-color:rgb(55,55,55);
}

QCheckBox:indicator:checked{
	background-color:green;
}</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="logging_maintabwidget_tab">
       <attribute name="title">
        <string>Status Logs</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>File Transfer Logs</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>File Cleanup Logs</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTextBrowser" name="cleanup_text_browser"/>
        </item>
        <item row="1" column="0">
         <widget class="QTextBrowser" name="file_transfer_text_browser"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settings_maintabwidget_tab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Folder Transfers</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="transfer_table_widget">
              <property name="styleSheet">
               <string notr="true">QHeaderView:section{
	background-color:rgb(55,55,55);
}</string>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <row>
               <property name="text">
                <string>1</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>Source Path</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Destination Path</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Schedule</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Cleanup</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Cleanup Age</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Enabled</string>
               </property>
              </column>
              <item row="0" column="0">
               <property name="text">
                <string>C:\Path\to\the\place\thats\good</string>
               </property>
              </item>
              <item row="0" column="1">
               <property name="text">
                <string>C:\Path\to\the\place\thats\good</string>
               </property>
              </item>
              <item row="0" column="2">
               <property name="text">
                <string>10:00 AM</string>
               </property>
              </item>
              <item row="0" column="3">
               <property name="text">
                <string>Yes</string>
               </property>
              </item>
              <item row="0" column="4">
               <property name="text">
                <string>90</string>
               </property>
              </item>
              <item row="0" column="5">
               <property name="text">
                <string>No</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Folder Cleanup</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="cleanup_table_widget">
              <property name="styleSheet">
               <string notr="true">QHeaderView:section{
	background-color:rgb(55,55,55);
}</string>
              </property>
              <property name="alternatingRowColors">
               <bool>false</bool>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <row>
               <property name="text">
                <string>1</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>Cleanup Path</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Check Path</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Schedule</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Cleanup Age</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Enabled</string>
               </property>
              </column>
              <item row="0" column="0">
               <property name="text">
                <string>C:\This\is\yet\another\path\to\places</string>
               </property>
              </item>
              <item row="0" column="1">
               <property name="text">
                <string>C:\This\is\yet\another\path\to\places</string>
               </property>
              </item>
              <item row="0" column="2">
               <property name="text">
                <string>4:00 PM</string>
               </property>
              </item>
              <item row="0" column="3">
               <property name="text">
                <string>180</string>
               </property>
              </item>
              <item row="0" column="4">
               <property name="text">
                <string>Yes</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Schedule</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="cleanup_schedule_time_edit"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Clean Source Once Files In Destination</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="transfer_clean_source_check_box">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Cleanup Age (Days)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="transfer_cleanup_age_spin_box">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximum">
               <number>365</number>
              </property>
              <property name="value">
               <number>365</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Schedule</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="transfer_schedule_time_edit">
              <property name="calendarPopup">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Cleanup Path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="cleanup_source_line_edit"/>
            </item>
            <item>
             <widget class="QPushButton" name="cleanup_source_browse_button">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="5" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QPushButton" name="cleanup_add_update_button">
              <property name="text">
               <string>Add/Update Queue</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cleanup_remove_button">
              <property name="text">
               <string>Remove From Queue</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_14">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Source Path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="transfer_source_line_edit">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="transfer_source_browse_button">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Enabled</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cleanup_enabled_check_box">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_11">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Verify Files Exist in Check Path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cleanup_verify_exist_check_box">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Cleanup Age (Days)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="cleanup_only_age_spin_box">
              <property name="maximum">
               <number>365</number>
              </property>
              <property name="value">
               <number>365</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QPushButton" name="transfer_add_update_button">
              <property name="text">
               <string>Add/Update Queue</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="transfer_remove_button">
              <property name="text">
               <string>Remove From Queue</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_13">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Destination Path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="transfer_destination_line_edit"/>
            </item>
            <item>
             <widget class="QPushButton" name="transfer_destination_browse_button">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Check Path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="check_path_line_edit"/>
            </item>
            <item>
             <widget class="QPushButton" name="check_path_browse_button">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Enabled</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="transfer_enabled_check_box">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
